compiler/hash-ast/src/ast.rs

```diff
@@ -567,7 +567,7 @@ define_tree! {
     #[node]
     pub struct TupleTy {
         /// inner types of the tuple type
-        pub entries: Children!(TyArg),
+        pub entries: Child!(TyParams),
```

Comment:
This should be Params no? `<ident> : <ty> = <expr>`

Reply:
But this is a tuple type right, this can only feature in a type position. Also, whilst it may use the `TyParam`, it doesn't allow for a default value to be specified as seen in the grammar: https://hash-org.github.io/hashc/features/types.html

Reply:
Yeah well, I guess it is not supported, but then it doesn't matter if Params or TyParams is used. However, if we ever decide to support it, it should be Params, not TyParams.

Reply:
Yeah you're right, perhaps we should change `FnTy` and `TupleTy` to use `Params`? However, we would need to introduce a `Params` node (but I think that is good for consistency).